### PR TITLE
Data Source: `fastly_services`

### PR DESCRIPTION
**Pull request by noseglid**
This adds a new datasource for listing service ids, and details about
services.

This can be used if you need to read the service configurations to
configure some other resource. For instance it could be useful to
combine this with a `fastly_service_authorization` to grant permissions
to all services in an account.

### CODE REVIEW

File: fastly/data_source_services.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+func dataSourceFastlyServices() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceFastlyServicesRead,
+
+		Schema: map[string]*schema.Schema{
+			"ids": {
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Description: "A list of ID of services in your account. This is limited to the services the API token can read.",
```
Comment (Integralist) [inline fastly/data_source_services.go]:
```suggestion
				Description: "A list of service IDs in your account. This is limited to the services the API token can read.",
```

**Review by Integralist — APPROVED** [review]
Looks good to me. Just one minor suggested edit.

**Comment by Integralist** [comment]
Thanks, @noseglid if you could run/commit changes from `make generate-docs` and then I can get this PR merged 👍🏻

**Comment by noseglid** [comment]
Ah of course. It's done now! 👍

**Comment by Integralist** [comment]
@noseglid This is now released:
https://github.com/fastly/terraform-provider-fastly/releases/tag/v2.2.0

**Comment by noseglid** [comment]
Fantastic, thank you 🙌

**Pull request merged**